backend/src/main/java/com/crm/dto/request/CustomerRequest.java

```diff
@@ -0,0 +1,24 @@
+package com.crm.dto.request;
+
+import com.crm.exception.ErrorDict;
+import lombok.*;
+
+import javax.validation.constraints.NotEmpty;
+
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+public class CustomerRequest {
+
+    @NotEmpty(message = ErrorDict.CUSTOMER_NAME_INVALID)
```

Comment:
I think that over String values would be better to use @ NotBlank annotation because NotEmpty allows blank spaces. More here: [https://www.baeldung.com/java-bean-validation-not-null-empty-blank](url)